.github/workflows/komodod_ci.yml

```diff
@@ -56,16 +65,192 @@ jobs:
 #           CONFIGURE_FLAGS='CPPFLAGS=-DTESTMODE' ./zcutil/build-mac.sh -j4
 #       - name: Run CC tests
 #         if: runner.os == 'Linux' || runner.os == 'macOS'
-      - name: Run CC tests
-        if: runner.os == 'Linux'
+
+
+  linux-test-maincc:
+    name: Test (Linux/mainCC)
```

Comment:
@SirSevenG Please change this one to some informative name e.g. "Test Dice, Token, Faucet, Rewards CC"
There is no such term as "mainCC"